### DISCUSSION TITLE
Question about fine tunning foundations model

### DISCUSSION THREAD

**Discussion by mahpe** [ACEsuit/mace#329] [Q&A]
Dear all

I am trying to fine tune the "large" foundation model for my data. However, when I run "run_train.py" the loss function for the model is not computed. 

I use the "foundation" branch and the below arguments:

  log_dir = "logs"
  model_dir = "."
  checkpoints_dir = "checkpoints"
  results_dir = "results"
  name="MACE_0"
  foundation_model="large"
  train_file="train.xyz"
  valid_fraction=0.05
  test_file="test.xyz"
  energy_weight=1.0
  forces_weight=10.0
  E0s="average"
  lr=0.01
  scaling="rms_forces_scaling"
  batch_size=2
  max_num_epochs = 150#1500
  ema=true
  ema_decay=0.99
  amsgrad=true
  default_dtype="float32"
  device="cuda"
  seed=3

Do you know why the loss is not calculated and evaluated for each batch?

**Comment by ilyes319** [comment]
Can you share the log file and training file please? What do you mean exactly by "the loss is not computed".

**Reply by mahpe** [reply]
Yes the log file is attached
[Test.log](https://github.com/ACEsuit/mace/files/14403086/Test.log)

**Comment by basillicus** [comment]
I think it may be related to this [issue](https://github.com/ACEsuit/mace/issues/412).

You may need to rename your labels in your train and test files to REF_energies and REF_forces, and use the flags --energy_key and --forces_key accordingly
